supervision/detection/line_zone.py

```diff
@@ -127,8 +137,82 @@ def out_count_per_class(self) -> Dict[int, int]:
         """
         return dict(self._out_count_per_class)
 
```

Comment:
`LineZone` `in_count` and `out_count` are documented in two places. I suggest dropping them from `Attributes`.

Reply:
Actually, no. PolygonZone uses attributes in primary docstring. I'll remove the property docstrings.